frontend/src/components/user-management/student/AddStudentModal.tsx

```diff
@@ -18,61 +19,49 @@ import {
   VStack,
 } from "@chakra-ui/react";
 
+import { CREATE_STUDENT } from "../../../APIClients/mutations/ClassMutations";
+import { StudentResponse } from "../../../APIClients/types/ClassClientTypes";
 import { PlusOutlineIcon } from "../../../assets/icons";
+import AuthContext from "../../../contexts/AuthContext";
 import { StudentForm, StudentInput } from "../../../types/ClassroomTypes";
 import ErrorToast from "../../common/ErrorToast";
 import ModalFooterButtons from "../../common/ModalFooterButtons";
+import Toast from "../../common/Toast";
 
 const AddStudentModal = (): React.ReactElement => {
   const {
+    handleSubmit,
     watch,
     setValue,
     formState: { errors },
   } = useFormContext<StudentForm>();
   const { onOpen, onClose, isOpen } = useDisclosure();
-  const [firstNameError, setFirstNameError] = React.useState(false);
-  const [lastNameError, setLastNameError] = React.useState(false);
-  const [studentNumberError, setStudentNumberError] = React.useState(false);
   const [showRequestError, setShowRequestError] = useState(false);
   const [requestErrorMessage, setRequestErrorMessage] = useState<string | null>(
     null,
   );
```

Comment:
i think you can replace lines 39 and 40 with `const [errorMessage, setErrorMessage] = useState("");`